### ISSUE TITLE
PhantomJS crashes when trying to create a new RegExp with / prefix

### ISSUE THREAD

**Issue by niemyjski** [ariya/phantomjs#13291]
```
ERROR [launcher]: PhantomJS crashed.
```

When I run a test that has the following code:

```js
var regex = new RegExp('/^test$');
```

PhantomJS doesn't crash if I do this:

```js
var regex = new RegExp('^test$');
```

**Comment by LeMoussel** [comment]
No crash with this config and the following code:
```javascript 
var regex = new RegExp('/^test$');
```

**Version of PhantomJS**
2.0.0

**Operating system**
Windows 7

**Did you use binary PhantomJS or did you compile it from source?**
    Binary

**Comment by niemyjski** [comment]
I found it while testing this [commit](https://github.com/exceptionless/Exceptionless.JavaScript/commit/30692c60cfed93cdc3bac18c6bccfec53b71334e).

I'm using the npm package version: "1.9.17",

**Comment by Vitallium** [comment]
Fixed in 2.0.
